.github/dependabot.yml

```diff
@@ -4,14 +4,54 @@ updates:
     directory: /
     schedule:
       interval: daily
+    # Create a group of dependencies to be updated together in one pull request
+    groups:
```

Comment:
Should a wildcard (*) be added in some group? I think this means you will miss grouped updates for any dependency does not match one of the patterns below. 

Is that intentional? The [docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups) aren't clear how the grouped updates will behave if a dependency matches a pattern in multiple groups. Or I missed it, which happens.

Reply:
Thinking about this more. You could probably get a little complicated here by having a wildcard group that has an exclude pattern for all the other patterns listed earlier.

Reply:
I'm not sure what you mean... here's an example of what sarama does https://github.com/IBM/sarama/blob/c10bd1e5709a7b47729b445bc98f2e41bc7cc0a8/.github/dependabot.yml#L27

Reply:
https://github.com/open-telemetry/opentelemetry-operator/pull/1807

I think what I am getting at is dependencies such as `github.com/stretchr/testify` and `github.com/gin-gonic/gin`  will not be matched into any group right? So this means we will still see one of PRs raised for these. That could be completely intentional, but the goal wasn't clear.

Reply:
I guess the PR title says `common things` and not `everything`. So looks good to me! Just adding some food for thought.

Reply:
oh yeah very intentional. I took a list of all the things I most frequently have to update and just did some grouping on those